src/hotspot/share/opto/memnode.hpp

```diff
@@ -292,6 +292,8 @@ class LoadNode : public MemNode {
   bool has_unknown_control_dependency() const  { return _control_dependency == UnknownControl; }
   bool has_pinned_control_dependency() const   { return _control_dependency == Pinned; }
 
+  LoadNode* pin_for_array_access() const override;
```

Comment:
Okay, apparently this does no build on macos (see GHA) due to `Winconsistent-missing-override`. So, you either have to mark all overriding methods in this class with `override` or use none at all. I guess it's simpler to just get ride of `override` here again.